src/website/views/templates/app-layout.pug

```diff
@@ -0,0 +1,944 @@
+---
+title: App layout
+description: A collection of application layout page templates built with Page Title, Buttons, Page-level actions and Footer.
+wide_container: true
+---
+
+.-lumen-only
+  h2 Examples
+  p.-text To render application layout, use the class <code>chi-main</code>.
+
+  h3 Base
+  .example.-mb--3
+    .chi-main
+      .chi-main__title.-no-back-link
+        .-d--flex.-flex--column
+          .-d--flex.-align-items--center
+            .-text--h3.-text--boldest.-text--navy.-m--0.-pr--2 App title
+      .chi-main__content
+        h1.-text--h4 App main content area
+        p.-text
+          | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
To eliminate any potential confusion where developers think the h1 or p tag are required, can we replace each example with the following:

```
  .-d--flex.-align-items--center.-justify-content--center(style="height:10rem;")
    | App content goes here
```

and then in the corresponding code snippet, remove the h1 and p element and replace with a comment
`<!-- App content goes here -->`